docs/training/apis-howto/api-pytorch-ug.rst

```diff
@@ -638,7 +636,271 @@ Take the following code for example.
            model = models.__dict__[args.arch]()
 
 ``args.arch`` is a hyperparameter. You should define the hyperparameter space in the
-:doc:`experiment configuration </reference/reference-training/experiment-config-reference>` and use
-``self.context.get_hparams()``, which gives you access to all the hyperparameters for the current
-trial. By doing so, you get better tracking in the WebUI, especially for experiments that use a
-searcher.
+:doc:`experiment configuration </reference/reference-training/experiment-config-reference>`. By
```

Comment:
do not use any `:doc:` references.  There are actually two duplicate link anchors at the top of that document, pick either of them.  But a `:doc:` reference hard-codes the current doc path into the content of the docs, which will break if we do any renaming (which Tara plans on doing a lot of).